### DISCUSSION TITLE
`go:embed` And Binaries

### DISCUSSION THREAD

**Discussion by PrashamTrivedi** [goreleaser/goreleaser#2223] [Q&A] [answered]
I have two binaries. One is embedded binary, and the other is main binary. Main binary looks for embedded binary in some path e.g. Relative path from binary.  How would I create both binaries?

I have to create two builds, which stores binaries in their mentioned ids. And they can't be same. How would I put first binary as relative path of second binary?

**Comment by caarlos0** [comment, marked as answer]
maybe you'll need to set `no_unique_dist_dir`

https://goreleaser.com/customization/build/
